ice_discharge.org

```diff
@@ -1801,6 +1778,24 @@ parallel "r.colors map=dh_{1} color=difference" ::: $(seq 1995 2019)
 #+END_SRC
 
 
+**** DEM
+
++ Merge Khan dh/dt w/ PRODEM to generate annual DEMs
+
+#+BEGIN_SRC bash :exports both :results verbatim
+MSG_OK "DEM"
+g.mapset -c DEM
+
+g.region raster=DEM_2020@PRODEM -pa
+for y in {2019..2022}; do
+  r.mapcalc "DEM_${y} = DEM_${y}@PRODEM"
+done
+
+for y in {2019..1995}; do
+  y1=$(( ${y} + 1 ))
+  r.mapcalc "DEM_${y} = DEM_${y1} - dh_${y}@Khan_2016"
```

Comment:
I don't understand why 2020 is the baseline DEM, when 2019 is the first PRODEM. With this line DEM_2019 is calculated from the Khan dh or am I missing something?